tools/constants.mjs

```diff
@@ -41,6 +41,10 @@ export const AOS_WASM = fs.readFileSync(
   ),
 );
 
+export const AOS_ANT_WASM = fs.readFileSync(
```

Comment:
Might want to add comment here about the importance of reading the file as binary data.

Reply:
[091b4fc](https://github.com/ar-io/ar-io-ant-process/pull/54/commits/091b4fcb8de8f2f1a342d2578d225f6af707f2c6)